homeassistant/components/fronius/sensor.py

```diff
@@ -145,18 +146,30 @@ def data(self):
         """Return the state attributes."""
         return self._fetched
 
+    @property
+    def available(self):
+        """Is the fronius device active?"""
+        return self._available
+
     async def async_update(self):
         """Retrieve and update latest state."""
         values = {}
```

Comment:
Then we can remove this too.

Reply:
```suggestion
```

Reply:
I think showing availability is not bad, this way the user sees when the device is online/offline. Or am I missing some implications of the attribute?